.gitignore

```diff
@@ -2,3 +2,4 @@
 literallycanvas/
 *.sublime-workspace
 node_modules/
+*.log
```

Comment:
NPM creates `.log` files when publishing, etc. so I'd recommend adding this to `.gitignore`